lib/src/layouts/multi_date.dart

```diff
@@ -100,7 +99,7 @@ class _MultiDateTimetableState<E extends Event>
           eventProvider: (visibleDates) =>
               eventProvider(visibleDates).where((it) => it.isPartDay).toList(),
           child: Builder(
-            builder: (contxt) => widget.contentBuilder(
+            builder: (context) => widget.contentBuilder(
```

Comment:
I noticed that typo by chance. Is it correct now or was it correct before and I should revert that change?

Reply:
Your fix is correct. The `Builder` providing the new context is so that the `contentBuilder` can call `DefaultEventProvider.of(context)` and receive the filtered events directly